spec/lib/mavenlink/model_spec.rb

```diff
@@ -540,4 +540,16 @@ def association_load_filters
       expect(subject.relatives.first["id"]).to eq("10")
     end
   end
+
+  describe "#to_h" do
+    subject { model.create(name: "Masha") }
+
+    it "return a hash with associated objects" do
+      expect(subject.to_h).to eq({
+                                   "name" => "Masha",
+                                   "id" => "7",
+                                   "relatives" => nil
```

Comment:
Does this mean that it will create a hash with the associated object even though the associated object does not exists?

Reply:
Yes